projects/website/markdown/posts/2021-09-21-tuist-2.0/post.mdx

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Releasing Tuist 2.0
+categories: ['Tuist', '2.0', 'Xcode', 'Swift', "Project generation"]
+excerpt: In this post, we share more details about this new major version of the project, 2.0, and present the direction we are taking as we move towards 3.0.
+author: pepibumur
+---
+
+After over [5 thousand commits](https://github.com/tuist/tuist/commits) from [120 contributors](https://github.com/tuist/tuist/graphs/contributors) and over three years after we landed the first commit on the project,
+I'm thrilled to share with all of you that we released the second major of the project, Tuist 2.0.
+
+As you might know,
+projects that follow [semantic versioning](https://semver.org/) use major versions to flag breaking changes in the release.
+From the user standpoint, that can be seen as unfavorable because the migration is usually manual.
+However,
+breaking changes are often necessary to continue to improve the developer experience (DX). 
+In this new major iteration of Tuist,
+we're achieving that **by moving away from poor API designs**, 
+replacing some of them with **more straightforward and convenient APIs**,
+and **pruning features** that were distant from the project's direction.
+
+Because the release already provides release notes and migration guidelines,
```

Comment:
Can we link to the notes / guide here?

Reply:
Yes! I was waiting for that part to be written to have a hyperlink I can refer to.